tests-e2e/core/snapshotenvironmentbinding_test.go

```diff
@@ -635,7 +637,7 @@ var _ = Describe("SnapshotEnvironmentBinding Reconciler E2E tests", func() {
 
 			err = k8s.Get(&gitopsDeployment, k8sClient)
 			Expect(err).To(BeNil())
-			Expect(gitopsDeployment.ObjectMeta.Labels["appstudio.openshift.io"]).ToNot(Equal("testing"))
+			Consistently(gitopsDeployment.ObjectMeta.Labels["appstudio.openshift.io"], "2m", "10s").ShouldNot(Equal("testing"))
```

Comment:
Likewise, the `gitopsDeployment` is fixed here. You likely want one of the functions from the fixture/binding package for this.

Reply:
Corrected. Even `Consistently` was not the right method to use here.